seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -163,11 +166,11 @@ def _reload_shared_variant_annotations(updated_variants_by_id, updated_families,
         for v in variant_models:
             variants_by_id[v.variant_id].append(v)
 
-        logger.info(f'Reloading shared annotations for {len(variant_models)} saved variants ({len(variants_by_id)} unique)')
+        logger.info(f'Reloading shared annotations for {len(variant_models)} {data_type} {genome_version} saved variants ({len(variants_by_id)} unique)')
 
         updated_variants_by_id = {
             variant_id: {k: v for k, v in variant.items() if k not in {'familyGuids', 'genotypes', 'genotypeFilters'}}
-            for variant_id, variant in updated_variants_by_id.items()
+            for variant_id, variant in (updated_variants_by_id or {}).items()
```

Comment:
I have a question about how this works in general (not necessarily related to your pr):

If `updated_variants_by_id` is empty, we won't update any shared annotations right? 
`bulk_update` is called on `updated_variant_models` which comes from `updated_variants_by_id`, so why don't call `bulk_update` on the variants in `variants_by_id` instead?

Reply:
looking at this more, it seems like `updated_variants_by_id` should be updated with the other variants in the `if fetch_variant_ids:` block